.circleci/config.yml

```diff
@@ -7,6 +7,12 @@ plain-go113: &plain-go113
       environment:
         GOPATH: "/home/circleci/go"
 
+.common: &common
+  tags: [ "runner:main", "size:large" ]
+
+variables:
```

Comment:
Why does this have to be global when only one job needs it? I don't think we even need the variable?